packages/platform/atoms/booker/BookerWebWrapper.tsx

```diff
@@ -88,6 +97,7 @@ export const BookerWebWrapper = (props: BookerWebWrapperAtomProps) => {
     hasSession,
     extraOptions: routerQuery,
     prefillFormParams,
+    prevResponse: prevResponse?.responses as Record<string, string>,
```

Comment:
`prevResponse?.responses` can be undefined as well which doesn't match the assertion used.